website/docs/FAQ.md

```diff
@@ -160,3 +160,13 @@ Otherwise, reply CONTINUE, or the reason why the task is not solved yet."""
 ```
 
 If you have problems with agents running `pip install` or get errors similar to `Error while fetching server API version: ('Connection aborted.', FileNotFoundError(2, 'No such file or directory')`, you can choose **'python:3'** as image as shown in the code example above and that should solve the problem.
+
+## ChromaDB fails in codespaces because of old version of sqlite3
+
+(from [issue #251](https://github.com/microsoft/autogen/issues/251))
```

Comment:
I think referring to the issue is good but maybe the text could briefly describe the issue so that the reader of GitHub doesn't have to navigate to the issue to understand the context.

Reply:
I added some more context to the FAQ 👍